src/languages/powershell.js

```diff
@@ -116,6 +133,13 @@ export default function(hljs){
     ]
   };
 
+  var PS_ALIAS = {
+    className: 'built_in',
+    variants: [
```

Comment:
I'm guessing the error is because you have no `begin` match here... but again I'm not sure why this is even needed and the keywords can't simply be added at the top level.